tanner/tests/test_session_analyzer.py

```diff
@@ -189,6 +192,6 @@ def test_find_location(self):
             country='United States',
```

Comment:
For fail case you can create separate mock which raises and exception instead of returning value